pkg/export/template.go

```diff
@@ -11,13 +11,18 @@ import (
 	"text/template"
 	"time"
 
+	"github.com/cli/cli/pkg/text"
 	"github.com/cli/cli/utils"
 	"github.com/mgutz/ansi"
 )
 
-func parseTemplate(tpl string, colorEnabled bool) (*template.Template, error) {
+func parseTemplate(w io.Writer, tpl string, colorEnabled bool) (*template.Template, error) {
```

Comment:
The addition of a writer to this function is a smell to me. Why would a method that parses a template require providing a writer for the product of template execution? Could we somehow work around having to add this?

Reply:
Originally I tried to avoid it, instead passing either a `bytes.Buffer` or `strings.Builder` but `text/tablewriter.Writer` wouldn't actually flush to the buffer. I tried both an initialized buffer and default nil (the internal `[]byte` buffer) but writing never filled. It seems tabwriter wasn't designed to work with buffers other than stdout. I can keep working at it if you prefer, but couldn't even find examples or discussion of anyone else attempting this.

Reply:
Now irrelevant. I've refactored a bit to use TablePrinter.